spec/yaml/swiftnav/sbp/imu.yaml

```diff
@@ -69,6 +69,127 @@ definitions:
+            units: 0.000000228406861 m/s^2
+            desc: Acceleration in the IMU frame X axis
+        - acc_y:
+            type: s32
+            units: 0.000000228406861 m/s^2
+            desc: Acceleration in the IMU frame Y axis
+        - acc_z:
+            type: s32
+            units: 0.000000228406861 m/s^2
+            desc: Acceleration in the IMU frame Z axis
+        - gyr_x:
+            type: s32
+            units: 0.000003725290298 deg/s
+            desc: Angular rate around IMU frame X axis
+        - gyr_y:
+            type: s32
+            units: 0.000003725290298 deg/s
+            desc: Angular rate around IMU frame Y axis
+        - gyr_z:
+            type: s32
+            units: 0.000003725290298 deg/s
+            desc: Angular rate around IMU frame Z axis
+        - imu_id:
+            type: u8
+            desc: Identifier of current IMU, e.g. to differentiate between multiple IMUs in the same vehicle
+        - flags:
+            type: u8
+            desc: Health and status flags
+            fields:
+              - 0:
+                  desc: IMU health indicator
+                  values:
+                      - 0: Healthy
+                      - 1: Not Healthy
+              - 1:
+                  desc: Accelerometer overrange indicator
+              - 2:
+                  desc: Gyroscope overrange indicator
+              - 3-6:
+                  desc: Reserved
+              - 7:
+                  desc: Timestamping mode
+                  values:
+                       - 0: Monotonic, local time
+                       - 1: GNSS time of week
+
+ - MSG_IMU_LOW_RATE_DATA:
+    id: 0x0906
+    short_desc: Low rate IMU data
+    desc: >
+      This message contains low-rate IMU meta-information like the type of the IMU that is being used,
+      temperature and extended status information.
+
+      The temperature field is a temperature difference from -60 degrees Celsius in 0.004 degree increments
+      - i.e. a value of 1 would mean a temperature of -59.996 degrees Celsius. An invalid or unavailable
+      temperature shall be denoted by a value of 0xFF
+
+      The flags field  contains high level status information just as the high rate IMU data but contains
+      additional space for more extensive status information. The usage of these reserved status bits depend
+      on the actual IMU model being used. Details can be found in the respective integration manual.
+    fields:
+        - imu_id:
+            type: u8
+            desc: Identifier of current IMU, e.g. to differentiate between multiple IMUs in the same vehicle
+        - imu_type:
+            type: u8
+            desc: IMU type used
+            values:
+              - 0: Bosch BMI160
+              - 1: ST Microelectronics ASM330LLH
+        - temperature:
+            type: u16
+            units: 0.004 K
+            desc: temperature difference from -60 degrees Celsius - use 0xFF for invalid
+        - flags:
```

Comment:
High rate data also contains a `flags` field with much the same items in it. Is it needed here as well?